docs/tutorials/helloworld.md

```diff
@@ -290,16 +290,19 @@ import TabItem from '@theme/TabItem';
 
 ## Testing Hello World
 
-Now we will test to see if everything works as expected.
+Now, to see if everything works as expected we can test starting a host in the editor. A host plays the role of a server and a client at the same time.
 
 1. Click **Play**.
-1. Click **Start Host** under **NetworkManager**. 
+1. Click the **Start Host** button in the **NetworkManager** GameObject's inspector.
+1. You should now see your scene with your Player Prefab spawned.
+
+You can also use the command line helper to launch a server and one or more client that will connect to the server. You should then see the plane on the server, until the first client connects. Then, for each connected client, a new Player Prefab will be spawned.
```

Comment:
Minor suggestion:

> You can also use the command line helper to launch a server and one or more clients to connect to the server. You should see the plane on the server until the first client connects. Then, a new Player Prefab will be spawned for each connected client.

What spawns the new Player Prefabs for each connected client?

Reply:
Good point, I'll clarify this